staging/src/k8s.io/legacy-cloud-providers/gce/nodeipam/config/v1alpha1/conversion.go

```diff
@@ -19,7 +19,7 @@ package v1alpha1
 import (
 	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/kube-controller-manager/config/v1alpha1"
```

Comment:
Is `kube-controller-manager` still where `v1alpha1.NodeIPAMControllerConfiguration` should belong? `cloud-provider-gcp` depends on `kube-controller-manager` so this works out mechanically.

@cheftako are there any non-obvious constraints that we should be considering here? I checked the dependency tree and didn't find anything obviously problematic.

(same thing applies to defaults.go, which depends on the same config)